org.opent2t.sample.hub.superpopular/org.opent2t.sample.hub.superpopular.json

```diff
@@ -0,0 +1,24 @@
+{
+  "id": "http://schemas.opentranslatorstothings.org/org.opent2t.sample.hub.superpopular#",
+  "$schema": "http://json-schema.org/draft-04/schema#",
+  "description": "",
+  "title": "OpenT2T Hub",
+  "definitions": {
+    "org.opent2t.sample.hub.superpopular": {
+      "type": "object",
+      "properties": {
+        "name": {
+          "description": "ReadOnly, Gets the name of the hub."
+        },
+        "devices": {
+          "description": "ReadOnly, Gets the current set of devices associated with the hub."
+        }
```

Comment:
I'm not an expert on OCF, but I think that might be the way to go.

I have a different issue with that though. Is the expectation that once we use this to enumerate devices, the caller then initializes the translator with only this information, and then the translator then needs to do the full request to, for example Wink, for every single device that is enumerated?

To go from no devices to a list of populated devices then requires a new network request for every single device to get the same information that Wink provides from the '/users/me/wink_devices' endpoint, which seems fairly inefficient to me.

Is there a reason that we don't just return a list of populated OCF devices here?

---
In reply to: [76335569](https://github.com/openT2T/translators/pull/75#discussion_r76335569) [](ancestors = 76335569,76334693,76154047)